package.json

```diff
@@ -243,6 +243,14 @@
           "type": "string",
           "markdownDescription": "Optional custom path to the .scalafix.conf file.\n\nShould be an absolute path and use forward slashes `/` for file separators (even on Windows)."
         },
+        "metals.scalafixRulesDependencies": {
+          "type": "array",
+          "scope": "machine",
+          "items": {
+            "type": "string"
+          },
+          "markdownDescription": "List of Scalafix rules dependencies in case they are not available by default in Metals. For example: `com.github.liancheng::organize-imports:0.6.0`, `::` indicates Scala dependency"
+        },
```

Comment:
I haven't looked at metals' PR yet so I have a few questions arisen.

> they are not available by default in Metals

How one can know if rule is available by default?

> For example: `com.github.liancheng::organize-imports:0.6.0`, `::` indicates Scala dependency

not sure how to understand the second part with `::`

Reply:
> How one can know if rule is available by default?
We should probably list them somewhere :thinking: 

> not sure how to understand the second part with ::

Maybe I tried exaplaining instead of just linking to the coursier convention.

Reply:
Listing known rules is a good idea.



> > not sure how to understand the second part with ::
> 
> Maybe I tried exaplaining instead of just linking to the coursier convention.

OMG, now I realized what you meant 😂  I read `Scala dependency` and then interpreted `::` a List member haha. It's probably better to provide link which explains what `::` in the context of dependencies is.

Reply:
I modified the explanation a bit. I will add the list of built in rules later on, since I haven't yet added and community ones,